bundled/tool/zenml_wrappers.py

```diff
@@ -675,3 +725,207 @@ def copy_stack(self, args) -> dict:
             self.StackComponentValidationError,
         ) as e:
             return {"error": str(e)}
+    
+    def create_stack(self, args: Tuple[str, Dict[str, str]]) -> Dict[str, str]:
+        """Creates a new ZenML Stack.
+
+        Args:
+            args (list): List containing the name and chosen components for the stack.
+        Returns:
+            Dictionary containing a message relevant to whether the action succeeded or failed
+        """
+        [name, components] = args
+
+        try:
+            self.client.create_stack(name, components)
+            return {"message": f"Stack {name} successfully created"}
+        except self.ZenMLBaseException as e:
+            return {"error": str(e)}
+        
+    def update_stack(self, args: Tuple[str, str, Dict[str, List[str]]]) -> Dict[str, str]:
+        """Updates a specified ZenML Stack.
+
+        Args:
+            args (list): List containing the id of the stack being updated, the new name, and the chosen components.
+        Returns:
+            Dictionary contianing a message relevant to whether the action succeeded or failed
+        """
+        [id, name, components] = args
+
+        try:
+            old = self.client.get_stack(id)
+            if old.name == name:
+                self.client.update_stack(name_id_or_prefix=id, component_updates=components)
+            else:
+                self.client.update_stack(name_id_or_prefix=id, name=name, component_updates=components)
+                
+            return {"message": f"Stack {name} successfully updated."}
+        except self.ZenMLBaseException as e:
+            return {"error": str(e)}
+        
+    def delete_stack(self, args: Tuple[str]) -> Dict[str, str]:
+        """Deletes a specified ZenML stack.
+
+        Args:
+            args (list): List contianing the id of the stack to delete.
+        Returns:
+            Dictionary contianing a message relevant to whether the action succeeded or failed
+        """
+        [id] = args
+
+        try:
+            self.client.delete_stack(id)
+
+            return {"message": f"Stack {id} successfully deleted."}
+        except self.ZenMLBaseException as e:
+            return {"error": str(e)}
+        
+    def create_component(self, args: Tuple[str, str, str, Dict[str, str]]) -> Dict[str, str]:
+        """Creates a new ZenML stack component.
+
+        Args:
+            args (list): List contianing the component type, flavor used, name, and configuration of the desired new component.
```

Comment:
```suggestion
            args (list): List containing the component type, flavor used, name, and configuration of the desired new component.
```

Reply:
Copy+pasting really did me a disservice, but I have fixed the typo